### DISCUSSION TITLE
Pip module ?

### DISCUSSION THREAD

**Discussion by u2gilles** [rdbende/Azure-ttk-theme#37] [Question] [answered]
Will you make a pip module for this theme as you did for sv_ttk ?

**Comment by rdbende** [comment, marked as answer]
Yes, it's in progress, and the forest theme too! Eventually I'll get to release them, but first I'd like to fix some bugs.

**Reply by u2gilles** [reply]
That's so great. I was considering learning other python GUIs to build apps with a look & feel similar to "material design" or "bootstrap CSS" but thanks to your themes, I don't need any more. "Tkinter + your themes" is just perfect for me for the time being.

**Comment by AntiApple4life** [comment]
Is the pip module still being worked on?
